packages/hash/frontend/src/pages/[account-slug]/entity-types/property-list-card.tsx

```diff
@@ -0,0 +1,701 @@
+import { faClose, faSearch } from "@fortawesome/free-solid-svg-icons";
+import { Button, ButtonProps } from "@hashintel/hash-design-system/button";
+import { Chip } from "@hashintel/hash-design-system/chip";
+import { FontAwesomeIcon } from "@hashintel/hash-design-system/fontawesome-icon";
+import { IconButton } from "@hashintel/hash-design-system/icon-button";
+import { TextField } from "@hashintel/hash-design-system/text-field";
+import { textFieldBorderRadius } from "@hashintel/hash-design-system/theme/components/inputs/mui-outlined-input-theme-options";
+import {
+  Autocomplete,
+  autocompleteClasses,
+  Box,
+  ButtonBase,
+  Checkbox,
+  checkboxClasses,
+  experimental_sx,
+  outlinedInputClasses,
+  Paper,
+  PaperProps,
+  PopperProps,
+  styled,
+  svgIconClasses,
+  Table,
+  TableBody,
+  tableBodyClasses,
+  TableCell,
+  tableCellClasses,
+  TableFooter,
+  TableHead,
+  TableRow,
+  tableRowClasses,
+  Theme,
+  Typography,
+  useForkRef,
+} from "@mui/material";
+import {
+  bindPopover,
+  bindToggle,
+  bindTrigger,
+  usePopupState,
+} from "material-ui-popup-state/hooks";
+import {
+  createContext,
+  Ref,
+  useContext,
+  useId,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+import { Modal } from "../../../components/Modals/Modal";
+import { ArrowUpRightIcon, CirclePlusIcon } from "../../../shared/icons/svg";
+import { PropertyTypeForm } from "./property-type-form";
+import { OntologyChip } from "./ontology-chip";
+import { PlaceholderIcon } from "./placeholder-icon";
+import { PropertyMenu } from "./property-menu";
+import { QuestionIcon } from "./question-icon";
+import { useStateCallback, withHandler } from "./util";
+import { WhiteCard } from "./white-card";
+
+const StyledPlusCircleIcon = styled(CirclePlusIcon)(
+  experimental_sx<Theme>({
+    height: "12px",
+  }),
+);
```

Comment:
We use `experimental_sx as sx` in everywhere else on our projects. This makes it inconsistent I think. Also no need to pass `Theme` afaik.

Reply:
I think renaming it sx is tricky when it can cause shadowing issues with eslint when you have other components in the same file which take an sx prop to make them styleable. Will remove the reference to theme (although it is necessary when using the theme – maybe we can make a version which passes the generic for us)